README.md

```diff
@@ -1,25 +1,148 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/Ruby_(programming_language)">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Setup and controllers**
+- **Controllers specs**
+- **Creating a data model**
+- **Processing data in models**
+- **Views**
+- **Forms**
+- **Validations, Model specs, and n+1 problems**
+- **Add Devise**
+- **Add authorization rules**
+- **Integration specs for Views**
+- **Add API endpoints**
+- **API documentation**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and use as a template, follow these steps.
+
+### Clone <a name="clone"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/edahigure/blog-app.git
+```
+
+To install the necessary dependencies, you should run:
+
+### `gem install bundler`
+
+### `bundler install`
+
+## 🤖 Available Scripts <a name="available-scripts"></a>
+
+In the project directory, you can run:
+
+### `rubocop -A`
+
+To fix all Linter error automatically
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="authors"></a>
+
+
+
+👤 **Edahi Antonio Gutiérrez Reyes**
+
+- GitHub: [@edahigure](https://github.com/edahigure)
+- Twitter: [@edahigure](https://twitter.com/edahigure)
+- LinkedIn: [edahigure](https://www.linkedin.com/in/edahigure/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **UI**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/edahigure/blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please let me know
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to microverse to asign me the homework
+<br/>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Is there any restriction to adapting the app to other porpuse??**
+
+  - No problem.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* Deployment instructions
 
-* ...
 # blog-app
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you remove this title at the bottom of your readme because you already have it at the top of your readme file so there is not point of having this title here 🤔 Kindly remove it to make your readme clean 👍